### DISCUSSION TITLE
Server Profiles disappeared

### DISCUSSION THREAD

**Discussion by ItsPike** [Foxlider/FASTER#220] [Q&A] [answered]
I recently opened faster and my server profiles are gone. I had a server running Antistasi Plus and I'm not sure how to get them back. So how do I get them back or at least create a new profile with the same server save? Dont really want to lose my groups antistasi progress. Thanks

**Comment by OverlordZorn** [comment, marked as answer]
https://github.com/Foxlider/FASTER/wiki/HowTo:-Restore-Faster-after-update

**Reply by OverlordZorn** [reply]
additionally: as long as you dont run an ancient version of antistasi, you should be able to find your antistasi progress as a missionprofile.vars file somewhere in the profile folder

https://community.bistudio.com/wiki/saveMissionProfileNamespace

**Reply by OverlordZorn** [reply]
![image](https://github.com/user-attachments/assets/87bd20cc-f452-4c6a-8d43-2d36502030bf)

**Comment by ItsPike** [comment]
Cheers thanks this is what I needed. Thanks a ton
________________________________
